mount.go

```diff
@@ -91,3 +95,56 @@ func getDeviceForPath(path string) (*device, error) {
 
 	return dev, nil
 }
+
+const (
+	procMountsFile = "/proc/mounts"
+
+	fieldsPerLine = 6
+)
+
+const (
+	procDeviceIndex = iota
+	procPathIndex
+	procTypeIndex
+)
+
+func getDevicePathAndFsType(mountPoint string) (devicePath, fsType string, err error) {
```

Comment:
sameo had a look at that package, there is one function that I can use for parsing /proc/mounts file as of now. I'd rather just do that, then import that package.